docs/registry/txt.md

```diff
@@ -199,3 +199,91 @@ The TXT registry can optionally cache DNS records read from the provider. This c
 rate limits imposed by the provider.
 
 Caching is enabled by specifying a cache duration with the `--txt-cache-interval` flag.
+
+## OwnerID migration
+
+If you wish to update the owner ID of the TXT records managed by your external-dns instance,
+you can set the following flags: `--migrate-txt-owner`, which will enable the migration checks
+in the run loop, `--txt-owner-id=new-owner-id` and `--from-txt-owner=old-owner-id`.
```

Comment:
```suggestion
The owner ID of the TXT records managed by external-dns instance can be updated.

When `--migrate-txt-owner` is set, it will enable the migration checks
in the run loop using `--txt-owner-id=new-owner-id` and `--from-txt-owner=old-owner-id`.
```